zerver/lib/users.py

```diff
@@ -61,15 +61,22 @@ def check_full_name(full_name_raw: str) -> str:
 # name (e.g. you can get there by reactivating a deactivated bot after
 # making a new bot with the same name).  This is just a check designed
 # to make it unlikely to happen by accident.
-def check_bot_name_available(realm_id: int, full_name: str) -> None:
+def check_bot_name_available(realm_id: int, full_name: str, is_activation: bool) -> None:
```

Comment:
Added an additional parameter to distinguish reactivating bots when calling the function.Updated the parameters in all function calls.